hs_file_types/tests/refts_method_link_unknown_uc.refts.json

```diff
@@ -0,0 +1,39 @@
+{ "timeSeriesReferenceFile" :
+	{ "referencedTimeSeries" : [
+		{
+			"beginDate" : "2016-04-06T00:00:00",
+            "endDate" : "2017-01-26T00:00:00+00:00",
+            "requestInfo": {
+				"networkName" : "NWISDV",
+				"refType" : "WOF",
+				"returnType" : "WaterML 1.1",
+				"serviceType" : "SOAP",
+				"url" : "http://hydroportal.cuahsi.org/nwisdv/cuahsi_1_1.asmx?WSDL"
+			},
+			"site": {
+				"siteName" : "HOBBLE CREEK AT 1650 WEST AT SPRINGVILLE, UTAH",
+				"siteCode" : "NWISDV:10153100",
+				"latitude" : 40.178871899999997,
+                "longitude" : -111.639338
+			},
+            "variable": {
+				"variableName" : "Discharge, cubic feet per second",
+				"variableCode" : "NWISDV:00060/DataType=MEAN"
+			},
+			"method" : {
+                "methodDescription" : "Estimated by NASA National Snow and Ice Data Center",
+                "methodLink" : "UNKNOWN"
+            },
+			"valueCount": 1000,
+			"sampleMedium" : "unknown"
```

Comment:
should there be a test for null as well?

Reply:
There is a test json file and a coresponding unit test for methodLink:""